koku/api/report/hcs/view.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright 2023 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""View for HCS report posting."""
```

Comment:
I probably wouldn't use HCS everywhere its just one (and current primary target for this.

Might call it something like simplified, or minimal cloud report ingestion.